exporter.py

```diff
@@ -17,17 +17,58 @@
 class HomematicMetricsProcessor(threading.Thread):
 
   METRICS_NAMESPACE = 'homematic'
-  DEFAULT_SUPPORTED_TYPES = [
-    'HmIP-SWO-PL',
-    'HmIP-STH',
-    'HmIP-STHD',
-    'HMIP-PSM',
-    'HmIP-FSM',
-    'HmIP-SWD',
-    'HMIP-SWDO',
-    'HM-WDS40-TH-I-2',
-    'HM-CC-RT-DN',
-    ]
+  # Supported Homematic (BidcosRF and IP) device types
+  # The list is the channel numbers where getParamset never works,
+  # or only sometimes works (e.g. if the device sent no data since
+  # the last CCU reboot).
+  DEFAULT_SUPPORTED_TYPES = {
```

Comment:
If you change the structure here, the init logic for a custom config and/or the sample config file must be changed, too. To keep the config understandable, I suggest to keep the list of supported devices as it is and add another structure, maybe a dict, with the invalid channels per device, where needed. Since only a few devices need those, it keeps the overhead smaller as well.

Reply:
I added a separate dictionary for the invalid channels.